styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -92,16 +92,16 @@
 }
 
 .hostingOptions-logo {
-  @include make-sm-column(5);
-  @include make-sm-column-push(3.5);
+  @include make-sm-column(3.5);
```

Comment:
I feel like the homepage is sort of a special case, since it needs to be so graphically wonky to work. Also, soon this section will go away and it wont be an issue anymore. Unfortunately right now, I don't really see another way to accomplish this?